ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -145,14 +145,37 @@ tests:
         IPv6=yes
         IPv4=yes
     workflow: baremetalds-assisted
-- as: e2e-metal-assisted-multi-version-suite
+- as: e2e-metal-assisted-4.6
   optional: true
   steps:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
-        OPENSHIFT_VERSION="all"
-      PACKET_PLAN: m2.xlarge.x86
+        OPENSHIFT_VERSION=4.6
+    workflow: baremetalds-assisted
+- as: e2e-metal-assisted-4.8
+  optional: true
```

Comment:
we can also make them totally required, it's good to do those complete checks on cherry-picks